rs/http_endpoints/public/src/common.rs

```diff
@@ -250,10 +250,15 @@ pub(crate) fn validation_error_to_http_error<C: std::fmt::Debug + HttpRequestCon
 ) -> HttpError {
     let message_id = request.id();
     match err {
-        RequestValidationError::InvalidSignature(_) => {
+        RequestValidationError::InvalidRequestExpiry(_)
+        | RequestValidationError::InvalidSignature(_) => {
+            let request = format!("{:?}", request);
             info!(
                 log,
-                "msg_id: {}, err: {}, request: {:?}", message_id, err, request
+                "msg_id: {}, err: {}, request: {}",
+                message_id,
+                err,
+                request[..request.len().min(1_024)].to_string()
```

Comment:
Do we expect any unicode characters here? Usually, it's unsafe to convert a slice to string as this could panic for it not being at [char boundary](https://doc.rust-lang.org/std/primitive.str.html#method.is_char_boundary).